discord.go

```diff
@@ -75,87 +55,8 @@ func New(args ...interface{}) (s *Session, err error) {
 	s.Identify.Properties.OS = runtime.GOOS
 	s.Identify.Properties.Browser = "DiscordGo v" + VERSION
 	s.Identify.Intents = MakeIntent(IntentsAllWithoutPrivileged)
```

Comment:
if this is being refactored anyways, `MakeIntent` is a no-op and not needed to be used anymore.

Reply:
Done.